src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -773,9 +787,26 @@ public void updateRecord(DsRecordDto record) throws Exception {
             log.error(message);
             throw new SQLException(message, e);
         }
-
     }
 
+
+    public void updateKalturaInternal(String kalturaReferenceId, String kalturaId) throws Exception {
```

Comment:
Please update the method name so that we know that it is about IDs.

Does "internal" refer to our internal DsStorage or the `KalturaInternalId` - I think this is a bit hard to understand from the method name. If it refers to the InternalId I would consider naming the method something like `updateKalturaInternalId` if it refers to internal as dsstorage i would prefer something like `updateKalturaIdsForRecord`